hack/00-osd-managed-cluster-config-integration.yaml.tmpl

```diff
@@ -1467,7 +1467,7 @@ objects:
       managed.openshift.io/gitHash: ${IMAGE_TAG}
       managed.openshift.io/gitRepoName: ${REPO_NAME}
       managed.openshift.io/osd: 'true'
-    name: layered-sre-authorization-addon-rhmi-operator
+    name: layered-sre-authorization-rhmi-addon-rhmi-operator
```

Comment:
Review how resourceApplyMode works.

Reply:
Specify "Sync" to allow deleting existing objects that were previously in the resources list.